src/main/resources/db/migration/common/V1_273__cleanup_arm_response_files_automated_task.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO darts.automated_task (aut_id,task_name,task_description,cron_expression,cron_editable)
+VALUES (10,'CleanupArmResponseFiles','Cleans up ARM response files','0 0 21 * * *',true);
+
+ALTER SEQUENCE aut_seq RESTART WITH 11;
```

Comment:
Wondering why we have a mixture of `darts` schema use in the same sql file - should the sequence also have the `darts.` prefix?